contracts/Aavegotchi/libraries/LibWhitelist.sol

```diff
@@ -5,6 +5,8 @@ import {LibMeta} from "../../shared/libraries/LibMeta.sol";
 import {LibAppStorage, AppStorage, Whitelist} from "../libraries/LibAppStorage.sol";
 
 library LibWhitelist {
+    event WhitelistAccessRightSet(uint32 indexed whitelistId, uint256 indexed actionRight, uint256 indexed accessRight);

```

Comment:
This event is never used @pakim249CAL

Reply:
@cinnabarhorse This event used in `setWhitelistAccessRight()`.